tests/Fixtures/frames/class.json

```diff
@@ -0,0 +1,5 @@
+{
+    "file": "path/to/file",
+    "line": 12,
+    "class": "CrashyClass"
```

Comment:
It really changes nothing, but to be coherent with the rest of the fixtures please use `TestClass` as class name

Reply:
Renamed.